safe-global/web-core
# Fix: contrastive logo for the safe mobile wallet

## What it solves
Resolves #546

## How this PR fixes it
Adds a new logo on a green bg.

## How to test it
Switch the Safe mobile app to dark mode. Connect to the app and check the logo on the device screen.
